### PR TITLE
Hw

### PR DESCRIPTION
**Pull request by IParov**

### CODE REVIEW

File: src/main/java/practice/CandidateValidator.java [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        int period = Arrays.stream(candidate.getPeriodsInUkr().split("-"))
+                .mapToInt(Integer::parseInt)
+                .reduce((a, b) -> Math.abs(a - b))
```
Comment (andriy-shulepa) [inline src/main/java/practice/CandidateValidator.java:14]: use more meaningful naming and try to get rid of Math.abs

File: src/main/java/practice/CandidateValidator.java [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+                .orElse(0);
+        return candidate.getAge() >= MIN_AGE
+                && candidate.isAllowedToVote() && candidate.getNationality().equals("Ukrainian")
+                && period >= 10;
```
Comment (andriy-shulepa) [inline src/main/java/practice/CandidateValidator.java:18]: extract magic number and string literal to constants

File: src/main/java/practice/StreamPractice.java [hunk 3]
```diff
@@ -14,7 +18,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
+                .boxed()
```
Comment (andriy-shulepa) [inline src/main/java/practice/StreamPractice.java:25]:
```suggestion
                .map(Integer::parseInt)
```

**Review by andriy-shulepa — CHANGES_REQUESTED** [review]

**Review by Rommelua — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-common-mistakes/java-core/java-eight-part-two/stream-api

Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected

**Pull request closed without merging**